modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/engine/VoskEngine.java

```diff
@@ -107,6 +107,8 @@ public Result generateSubtitlesFile(File mediaFile, File workingDirectory,
     }
 
     var output = new File(workingDirectory, FilenameUtils.getBaseName(mediaFile.getAbsolutePath()));
```

Comment:
Could it be that just adding the extension to the output file is enough? It seems like webvtt-py adds the extension if it's missing.

https://github.com/glut23/webvtt-py/blob/2f78325d96f260540fe4ba729d3624daaf3f9be6/webvtt/webvtt.py#L87-L90

Reply:
Yep, this works better.  One moment please...